prboom2/src/umapinfo.cpp

```diff
@@ -630,7 +630,6 @@ int ParseUMapInfo(const unsigned char *buffer, size_t length, umapinfo_errorfunc
 		if (parsed.endpic[0] && (strcmp(parsed.endpic, "-") != 0))
 		{
 			parsed.nextmap[0] = parsed.nextsecret[0] = 0;
-			if (parsed.endpic[0] == '!') parsed.endpic[0] = 0;
```

Comment:
Why was this sucker line there in the first place?

Reply:
Eh, I don't know, just a bug.